app/views/provider_interface/provider_agreements/data_sharing_agreement.html.erb

```diff
@@ -350,17 +346,16 @@
       </ul>
       <p>In the event of a significant security breach or other serious breach of the terms of this DSA by either participant, the DSA will be terminated or suspended immediately without notice.</p>
 
-      <% if !@provider_agreement.persisted? %>
-        <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-top-6 govuk-!-margin-bottom-6">
-        <%= form_with model: @provider_agreement, url: provider_interface_create_data_sharing_agreement_path, html: { novalidate: false } do |f| %>
-          <%= f.govuk_error_summary %>
-          <%= f.hidden_field :agreement_type %>
-          <%= f.hidden_field :provider_id %>
-          <% declaration = "#{@provider_agreement.provider.name} agrees to comply with the data sharing practices outlined in this agreement" %>
-          <%= f.govuk_check_box :accept_agreement, true, multiple: false, link_errors: true, label: { text: declaration } %>
-          <%= f.govuk_submit 'Continue' %>
-        <% end %>
-      <% end %>
+      <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-top-6 govuk-!-margin-bottom-6">
+
+      <% declaration = "#{@provider_agreement.provider.name} agrees to comply with the data sharing practices outlined in this agreement" %>
+      <%= f.govuk_check_box :accept_agreement, true, multiple: false, link_errors: true, label: { text: declaration } %>
```

Comment:
The traditional red highlight doesn't appear when this is missing and the error at the top appears — maybe it needs a `fieldset`?